ansible/library/exabgp.py

```diff
@@ -140,10 +140,24 @@ def start_exabgp(module, name):
     refresh_supervisord(module)
     exec_command(module, cmd="supervisorctl start exabgp-%s" % name)
 
+    for count in range(0, 5):
+        time.sleep(15)
```

Comment:
you can do more frequently, per 1 second, and wait for 60 seconds.

Reply:
ACK